vlib/vweb/README.md

```diff
@@ -61,3 +61,77 @@ That means that the template automatically has access to that action's entire en
 ### Deploying vweb apps
 
 Everything, including HTML templates, is in one binary file. That's all you need to deploy.
+
+## Getting Started
+
+To start with vweb, you have to import the module `vweb` first.
+After it, it's recommended to define a struct to access `vweb.Context`.
+The web server can be started by calling `vweb.run<App>(port)`.
+A complete
+
+**Example:**
+```v ignore
+import vweb
+
+struct App {
+    vweb.Context
+}
+
+fn main() {
+	vweb.run<App>(8080)
+}
+```
+
+### Defining endpoints
+To add endpoints to your web server, you have to extend the `App` struct.
+The function names are automatically mapped to the URL path and a return 
+value of `vweb.Result` is expected.
+
+**Example:**
+```v ignore
+fn (mut app App) hello() vweb.Result {
+	return app.text('Hello')
+}
+```
+
+This endpoint can be accessed via `http://localhost:port/hello`.
+
+To create an HTTP POST endpoint, you simply add a `[post]` attribute before the function definition.
+
+**Example:**
+```v ignore
+[post]
+fn (mut app App) world() vweb.Result {
+	return app.text('World')
+}
+```
+
+Instead of using the auto-mapping, you can 
```

Comment:
Instead of using auto-mapping, you can

Reply:
it is not mapping, it is routing

Reply:
mapping in this context will be confusing imho

Reply:
Function names are mapped to the routing. If you use the attribute to define a specific path, then you could talk about routing.
I could describe this point better.